### PR TITLE
make ratpm25pm10 usable @ aeroval

### LINKED ISSUE #1030: make the variable ratpm25pm10 usable
The new really needed variable is now the ration between `concpm25` and `concpm10` instead of the other way round.

Make the variable `ratpm25pm10` usable:
- adjust ranges (0 to 1 instead of the current -150 to +150)
- add proper color ranges to aeroval
- add proper naming to aeroval

### PR DESCRIPTION
**Pull request by jgriesfeller**
adding some definitions and adjusting limits

### CODE REVIEW

**Comment by jgriesfeller** [comment]
Commment: testing would require a full blown aeroval run since the obs data is calculated during runtime by aeroval. Only model data exists.

**Comment by jgriesfeller** [comment]
documentation has been added to user doc Google Doc file: https://docs.google.com/document/d/1LcAsJTDtlQFTlPQAtfllVLNwKW0SDbZwXd6uLrbbYrQ/edit?usp=sharing

File: pyaerocom/data/variables.ini [hunk 1]
```diff
@@ -5545,15 +5545,15 @@ comments_and_purpose = For physics evaluation and chemistry interpretation above
 [ratpm10pm25]
 description = ratio of pm10 and pm25
 unit = 1
-minimum = -150
-maximum = 150
+minimum = 0
+maximum = 10
```
Comment (lewisblake) [inline pyaerocom/data/variables.ini:5549]: Shouldn't this be 1 as well?

**Review by lewisblake — COMMENTED** [review]
Small question

File: pyaerocom/data/variables.ini (hunk 1 above)
Comment (jgriesfeller) [reply pyaerocom/data/variables.ini:5549]: nope, concpm10 is bigger than concpm25

**Review by lewisblake — APPROVED** [review]

**Pull request merged**